File: app/src/main/java/com/worldventures/dreamtrips/wallet/service/nxt/model/MultiResponseElement.java
package com.worldventures.dreamtrips.wallet.service.nxt.model;

import android.support.annotation.Nullable;

import com.google.gson.annotations.SerializedName;

import org.immutables.gson.Gson;
import org.immutables.value.Value;

@Gson.TypeAdapters
@Value.Immutable
public interface MultiResponseElement {

   @SerializedName("RefId")
   String referenceId();

   @Nullable
   @SerializedName("Value")
   String value();

   @Nullable
   @SerializedName("Error")
   MultiErrorResponse error();

}

File: wallet/src/main/java/com/worldventures/wallet/service/nxt/model/MultiResponseElement.kt
package com.worldventures.wallet.service.nxt.model

import com.google.gson.annotations.SerializedName

data class MultiResponseElement(
      @field:SerializedName("RefId") val referenceId: String,
      @field:SerializedName("Value") val value: String? = null,
      @field:SerializedName("Error") val error: MultiErrorResponse? = null)